modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Options/util.es.js

```diff
@@ -105,9 +105,8 @@ export const dedupValue = (fields, value, id) => {
  * 1. If the current value is null, use the label
  * 2. If the current label is null, use the string Option
  */
-export const normalizeValue = (fields, currentField) => {
-	const {label, value: prevValue} = currentField;
-	let value = prevValue ? prevValue : label;
+export const normalizeValue = (fields, currentValue, currentField) => {
```

Comment:
@carolmariaabb, can we add `currentValue` to be after `currentField` due to the alphabetic order?